src/AccessibilityInsights.SharedUx/ViewModels/RuleResultViewModel.cs

```diff
@@ -180,7 +185,7 @@ public BugInformation GetBugInformation()
                 elementPath: string.Join("<br/>", this.Element.GetPathFromOriginAncestor().Select(el => el.Glimpse)),
                 testMessages: string.Join("<br/>", this.RuleResult.Messages),
```

Comment:
Do we need to format this, not sure but in the ADO bug it doesn't look like how it looks in the AI-Win App, we may need to sync up

Reply:
Let's talk when everyone is in.

Reply:
as discussed, out of scope, later

Reply:
We had a chat offline. This is how it happens today as well. We will punt on the decision as to whether we want to fix this as a part of the feature.